src/OrleansServiceBus/Providers/Streams/EventHub/IEventHubSettings.cs

```diff
@@ -8,6 +8,13 @@ public interface IEventHubSettings
         string ConsumerGroup { get; }
         string Path { get; }
         int? PrefetchCount { get; }
+
+        /// <summary>
+        /// Indicates if stream provider should read all new data in partition, or from start of partition.
+        /// True - read all new data added to partition.
+        /// False - start reading from beginning of partition.
+        /// Note: If checkpoints are used, stream provider will always begin reading from most recent checkpoint.
+        /// </summary>
         bool StartFromNow { get; }
```

Comment:
Agreed.  I've played with the idea of using enums or enum flags, but not yet convinced myself that the alternatives are better.

Open to suggestions if there are alternatives that are arguably better.  A change to this would be better as a separate PR though, so if you have suggestions, please post and if better approach is settled on, I’ll submit in separate PR.